### PR TITLE
bug 1032227: fix sorting of hex addresses in signature report

### PR DESCRIPTION
**Pull request by willkg**
Hex addresses were getting sorted alphanumerically. This fixes the address value so that it gets converted to a base 10 int and then sorted.

### CODE REVIEW

**Comment by willkg** [comment]
@rehandalal: Can you look this over for "Oh dear--don't do that in JS." kinds of things?

File: webapp-django/crashstats/crashstats/static/crashstats/js/socorro/tablesorter_utils.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+(function ($) {
+  $.tablesorter.addParser({
+    id: 'hexdigit',
+    is: function () {
+      return false;
+    },
+    format: function (str) {
+      var newstr;
+
+      if (str == null || str == '') {
```
Comment (rehandalal) [inline webapp-django/crashstats/crashstats/static/crashstats/js/socorro/tablesorter_utils.js:10]: nit: use `===` especially with `null` check

File: webapp-django/crashstats/crashstats/static/crashstats/js/socorro/tablesorter_utils.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        return 0;
+      }
+
+      // if it starts with 0x, peel that off and if all digits, convert the
+      // rest from base 16 to base 10
+      if (str.substring(0, 2) == '0x') {
```
Comment (rehandalal) [inline webapp-django/crashstats/crashstats/static/crashstats/js/socorro/tablesorter_utils.js:16]: `===`

**Review by rehandalal — APPROVED** [review]
lgtm. just a couple little nits

File: webapp-django/crashstats/crashstats/static/crashstats/js/socorro/tablesorter_utils.js (hunk 1 above)
Comment (willkg) [reply webapp-django/crashstats/crashstats/static/crashstats/js/socorro/tablesorter_utils.js]: Oo--thank you! I'm surprised that this wasn't picked up in linting. Maybe I've got linting set up wrong.

**Comment by willkg** [comment]
Thank you!

**Pull request merged**